lib/src/clojure_lsp/handlers.clj

```diff
@@ -275,7 +275,7 @@
      :client-settings (:client-settings db-value)
      :final-settings (settings/all db-value)
      :cljfmt-raw (binding [*print-meta* true]
-                   (with-out-str (pr (f.format/resolve-user-cljfmt-config db-value))))
+                   (pr-str (f.format/resolve-user-cljfmt-config db-value)))
```

Comment:
Equivalent, but more idiomatic. Found during a post-mortem of #1080, while I was looking for places we might accidentally print to stdout.